flexget/plugins/input/next_trakt_episodes.py

```diff
@@ -75,6 +75,7 @@ def on_task_input(self, task, config):
                     trakt_id = item['show']['ids']['trakt']
                     listed_series[trakt_id] = {
                         'series_name': '%s (%s)' % (item['show']['title'], item['show']['year']),
+                        'series_year': item['show']['year'],
```

Comment:
> why should it not also fill in the series_year, so those two fields are consistent.

Because it seems to imply that the series plugin has some sort of support for series_year, when it doesn't. I think it would be confusing to see that field populated for your series when you enable the trakt_lookup plugin, but not have it be there otherwise. When you enable the trakt_lookup plugin, you can explicitly access the year via trakt_series_year.